doc/lsst.ap.pipe/apdb.rst

```diff
@@ -46,23 +45,26 @@ Databases can be configured using direct config overrides (see :ref:`ap-pipe-pip
 
 .. prompt:: bash
 
-   make_apdb.py -c diaPipe.apdb.isolation_level=READ_UNCOMMITTED diaPipe.apdb.db_url="sqlite:///databases/apdb.db" differencer.coaddName=dcr
+   make_apdb.py -c isolation_level=READ_UNCOMMITTED db_url="sqlite:///databases/apdb.db"
    ap_pipe.py -c diaPipe.apdb.isolation_level=READ_UNCOMMITTED diaPipe.apdb.db_url="sqlite:///databases/apdb.db" differencer.coaddName=dcr repo --calib repo/calibs --rerun myrun --id [optional IDs to process]
 
-|make_apdb| ignores any `ApPipeConfig` fields not related to the APDB (in the example, ``differencer.coaddName``), so there is no need to filter them out.
+The user is responsible for making sure the two APDB configurations match.
 
 In Gen 3, this becomes (see :ref:`ap-pipe-pipeline-tutorial` for an explanation of |pipetask|):
 
 .. prompt:: bash
 
-   make_apdb.py -c diaPipe.apdb.isolation_level=READ_UNCOMMITTED diaPipe.apdb.db_url="sqlite:///databases/apdb.db" differencer.coaddName=dcr
+   make_apdb.py -c isolation_level=READ_UNCOMMITTED db_url="sqlite:///databases/apdb.db"
    pipetask run -p ApPipe.yaml -c diaPipe:apdb.isolation_level=READ_UNCOMMITTED diaPipe:apdb.db_url="sqlite:///databases/apdb.db" differencer:coaddName=dcr -b repo -o myrun
 
+Again, make sure the APDB is created and accessed with the same configuration; note that the pipeline file given by ``-p`` may include APDB config overrides.
+You can double-check what configuration is being run by calling :command:`pipetask run` with the ``--show config="apdb*"`` argument, though this lists *all* configuration options, including those left at their defaults.
+
 Databases can also be set up using config files:
 
 .. prompt:: bash
 
-   make_apdb.py -C myApPipeConfig.py
+   make_apdb.py -C myApdbConfig.py
```

Comment:
It would be the same as the inline examples above, but with an extra `config.` in front. Could we just point people to the general config file documentation?

Reply:
I'm inclined to have a quote-text-box (essentially `cat myApdbConfig.py`) here *and also* link to the config documentation, if it's not too much trouble. `make_apdb.py` is an exception in so many other ways that users might not assume it uses any kind of standard configs either, and a quick example could enhance clarity.